docs/api/rest_api/rest_api_usage/rest_requests.md

```diff
@@ -136,6 +137,14 @@ Examples of such requests are:
 - [moving a Location and its subtree](../rest_api_reference/rest_api_reference.html#managing-content-move-subtree)
 - [swapping a Location with another](../rest_api_reference/rest_api_reference.html#managing-content-swap-location)
 
+### Expected user
+
+The `X-Expected-User` header specifies the user needed on client side for the request execution.
```

Comment:
This part of the sentence does not convey any information and might be confusing.
```suggestion
The `X-Expected-User` header specifies the user needed for the request execution.
```